davincipas/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+org = "healthcare"
+name = "davincipas"
+version = "1.0.0"
+distribution = "2201.9.0"
+authors = ["Ballerina"]
+keywords = ["Healthcare", "FHIR", "R4", "davincipas"]
+export = ["davincipas"]
+
+[[dependency]]
+org = "ballerinax"
+name = "health.fhir.r4"
+version = "6.0.1"
```

Comment:
Let's remove the dependencies from Ballerina toml to resolve it from the bal compiler itself.